### PR TITLE
Add ability to specify chart generation url for self hosted QuickChart

### PR DESCRIPTION
**Pull request by dianaduca**

### CODE REVIEW

File: QuickChart/QuickChart.cs [hunk 1]
```diff
@@ -31,17 +31,26 @@ public class Chart
...
             Width = 500;
             Height = 300;
             DevicePixelRatio = 1.0;
             Format = "png";
             BackgroundColor = "transparent";
 
-            Scheme = "https";
-            Host = "quickchart.io";
-            Port = 443;
+            if (scheme != null && host != null && port.HasValue)
```
Comment (rhumborl) [inline QuickChart/QuickChart.cs]: Maybe it should be a bit cleverer, e.g.

- if they specify a host only, default to https:443 (move host to first parameter, so they can do `new Chart("example.com")`)
- host and http, set port to 80
- host and https, set port to 443

and add relevant tests

Comment (dianaduca) [reply QuickChart/QuickChart.cs]: Updated

**Comment by typpo** [comment]
Thank you!

**Comment by dianaduca** [comment]
Could you please deploy a new version of the nuget package, including this work? Thanks

**Comment by typpo** [comment]
Released in 2.3.0 :)

**Comment by dianaduca** [comment]
Thank you !

**Pull request merged**